### PR TITLE
Solve PS 78

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
Solve the issue in which the dialog still appears when the user come back.

### CODE REVIEW

**Review by kiroj305 — CHANGES_REQUESTED** [review]
@kjawadDeveloper2 , I tried the `fix/ps_78` branch. It's occurring an error (Username not found) while fetching the wallet profile, though I already logged in the Wallet. What's the issue?

File: lib/screens/routing_screen.dart [hunk 1]
```diff
@@ -21,85 +23,100 @@ class RoutingScreen extends StatefulWidget {
...
-              children: [
-                ValueListenableBuilder<String>(
-                    valueListenable: username,
-                    builder: (_, String name, __) {
-                      return Text("$kWelcomeToEaselText, $name");
-                    }),
-              ],
+    return FocusDetector(
+      onFocusGained: () async {
+        print("OnFocusFained again");
```
Comment (mustafapylons) [inline lib/screens/routing_screen.dart]: typo to fix, "OnFocusGained again"

**Review by mustafapylons — COMMENTED** [review]
One small typo, otherwise good code.

**Comment by kjawadDeveloper2** [comment]
@Tiny-Angel305  this fetching is because of the grpc url change. I have added another PR which is already approved and merge. Can you check with the new wallet code?

**Comment by kiroj305** [comment]
@kjawadDeveloper2 , just checked this PR again along with the latest wallet main branch. No luck yet tho. Easel keeps trying to fetch the profile again and again.

**Review by kiroj305 — APPROVED** [review]

**Pull request merged**